spec/imgix/rails_spec.rb

```diff
@@ -255,7 +248,7 @@
 
         it 'generates the expected number of srcset values' do
           tag = Nokogiri::HTML.fragment(helper.ix_image_tag("image.jpg")).children[0]
-          expect(tag.attribute('srcset').value.split(',').size).to eq(71)
+          expect(tag.attribute('srcset').value.split(',').size).to eq(28)
```

Comment:
I realised I didn't mention this in my document... do the generated `srcsets` have a fallback src? It's a good idea to make sure they do. 

See this line in react-imgix https://github.com/imgix/react-imgix/blob/master/src/react-imgix.js#L84

A fallback image goes at the end of the srcSet list, and has no width descriptor. It should go something like `srcSet="url 100w, url 200w, url"`